### PR TITLE
feat: sb docs overview Avartar.tsx

### LINKED ISSUE #36: Storybook Data display docs overview missing

### PR DESCRIPTION
**Pull request by Mdjuealhossain**

### CODE REVIEW

**Comment by Mdjuealhossain** [comment]
can I write FormControl docs overview in [36-storybook-data-display-docs-overview-missing] branch...???

**Comment by Mdjuealhossain** [comment]
FormControl-missing Des

File: docs/Badge.stories.tsx [hunk 1]
```diff
@@ -11,8 +11,18 @@ const meta: Meta<typeof Badge> = {
     docs: {
       source: { language: "tsx", format: true, type: "dynamic" },
       description: {
-        component:
-          "The value must be chosen from a predefined set of allowed values.",
+        component: overView({
+          override: "Badge",
+          description: "Badge generates a small badge to the top-right of its child(ren).",
```
Comment (arifulbgt4) [inline docs/Badge.stories.tsx]: Change the text, child(ren) = children

Comment (Mdjuealhossain) [reply docs/Badge.stories.tsx]: solve

**Comment by arifulbgt4** [comment]
> 



> 

Sure

**Review by arifulbgt4 — APPROVED** [review]

**Pull request merged**